arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -99,3 +138,55 @@ def test_inference_data_attrs(posterior, prior, save_warmup, warmup_iterations:
     fails = check_multiple_attrs(test_dict, arviz_inference_data_from_pyjags_samples_dict)
     print(fails)
     assert not fails
+
+
+@pytest.fixture()
+def jags_prior_model() -> pyjags.Model:
+    return pyjags.Model(
```

Comment:
It would help others if `pyjags.Model(` was assigned to a var, and that var was returned, because it makes it possible to insert a debugger and understand what is being generated with `pyjags.Model(`. Returning the computed value immediately makes it harder to interrupt the execution flow and introspect

Reply:
Ok, I pushed these changes.